compiler/z/codegen/S390Instruction.cpp

```diff
@@ -2041,29 +2086,6 @@ TR::S390RILInstruction::generateBinaryEncoding()
       }
    else if (getOpCode().getOpCodeValue() == TR::InstOpCode::LARL)
       {
-      // Check if this LARL is loading unloadable constant
```

Comment:
Confirming, we would not be using `LARL` where it is used for static PIC sites or method sites at all, right?

Reply:
With https://github.com/eclipse-openj9/openj9/pull/14932 all uses of `LARL` for PIC sites will be removed, *except* for a usage in OMR (`aloadHelper()`) that I just found while double checking the codebase. I've amended my commit to leave this code in.